recipe/gen_patch_json.py

```diff
@@ -628,6 +628,19 @@ def _gen_new_index(repodata, subdir):
             new_constrains.append("sysroot_" + subdir + " ==99999999999")
             record["constrains"] = new_constrains
 
+        if (
+            subdir in ["linux-64", "linux-aarch64", "linux-ppc64le"]
+            and record_name in [
+                "gcc_impl_" + subdir, "gxx_impl_" + subdir, "gfortran_impl_" + subdir]
+            and record['version'] in ['5.4.0', '7.2.0', '7.3.0', '8.2.0']
+        ):
+            new_constrains = record.get('constrains', [])
+            for pkg in ["libgcc-ng", "libstdcxx-ng", "libgfortran", "libgomp"]:
+                new_constrains.append("{} 5.4.*|7.2.*|7.3.*|8.2.*|9.1.*|9.2.*".format(pkg))
+            new_constrains.append("binutils_impl_" + subdir + " <2.34")
+            new_constrains.append("ld_impl_" + subdir + " <2.34")
```

Comment:
Why these?

Reply:
It's the same as https://github.com/conda-forge/binutils-feedstock/blob/master/recipe/meta.yaml#L42-L44, but I'd like to remove the constraints from the recipe there and have it here.

Reply:
ahhhhh thank you